components/MadeWithPrezly/MadeWithPrezly.tsx

```diff
@@ -5,7 +5,7 @@ import styles from './MadeWithPrezly.module.scss';
 export function MadeWithPrezly() {
     return (
         <a
-            href="https://prez.ly/storytelling-platform"
+            href="https://prezly.com"
```

Comment:
Should we maybe add some UTM parameters?

Reply:
I'm waiting for a response [in linear](https://linear.app/prezly/issue/DEV-19834/change-powered-by-prezly-in-bea-to-point-to-instead-of-featureall-in#comment-719d6836). I will merge this one as is, and add param if requested.